BankApplication/src/com/luxoft/bankapp/comands/FindClientCommand.java

```diff
@@ -0,0 +1,29 @@
+package com.luxoft.bankapp.comands;
+
+import java.util.Scanner;
+
+import com.luxoft.bankapp.service.BankService;
+import com.luxoft.bankapp.service.BankServiceImpl;
+
+public class FindClientCommand implements Command {
+
+	@Override
+	public void execute() {
+		Scanner scan=new Scanner(System.in);
+		System.out.println("Enter Client name");
+		String clientName=scan.nextLine();
+		BankService bankService= new BankServiceImpl();
+		BankCommander.currentClient=bankService.getClient(BankCommander.currentBank, clientName);
+		if(BankCommander.currentClient!=null)
+			System.out.println(BankCommander.currentClient.getName() + " founded!");
```

Comment:
Just an English mistake: should be "found".

Founded - it's like you established the company, it's not result of the search.